django/db/migrations/state.py

```diff
@@ -443,6 +443,10 @@ def from_model(cls, model, exclude_rels=False):
             for key in ["unique_together", "index_together", "order_with_respect_to"]:
                 if key in options:
                     del options[key]
+        # Since we're completely ignoring private fields we make sure to
```

Comment:
I think the language is simpler without "we", e.g. "Private fields are ignored, so remove options that refer to them."